src/api/app/controllers/webui/cloud/azure/upload_jobs_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Webui
+  module Cloud
+    module Azure
+      class UploadJobsController < ::Webui::Cloud::UploadJobsController
+        def new
+          xml_object = OpenStruct.new(params.slice(:project, :package, :repository, :arch, :filename))
+          @upload_job = ::Cloud::Backend::UploadJob.new(xml_object: xml_object)
+          @crumb_list.push << 'Azure'
+        end
+
+        def create
```

Comment:
Now it is the same as the ec2 create action and you can move it back to it's original place in ```src/api/app/controllers/webui/cloud/upload_jobs_controller.rb```

Reply:
Done